### PR TITLE
- bugfixes

### PR DESCRIPTION
**Pull request by jgangemi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -187,6 +188,10 @@
       <artifactId>maven-resolver-impl</artifactId>
       <version>${maven-resolver.version}</version>
       <exclusions>
+        <exclusion>
```
Comment (stifm) [inline pom.xml:191]: This exclusion probably caused our build to fail, so we had to revert back to version 2.0.2:
`java.lang.NoClassDefFoundError: org/apache/commons/lang3/StringUtils
	at org.eclipse.aether.internal.impl.DefaultRemoteRepositoryManager.merge(DefaultRemoteRepositoryManager.java:368)
	at org.eclipse.aether.internal.impl.DefaultRemoteRepositoryManager.merge(DefaultRemoteRepositoryManager.java:290)
	at org.eclipse.aether.internal.impl.DefaultRemoteRepositoryManager.getPolicy(DefaultRemoteRepositoryManager.java:278)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.gatherDownloads(DefaultArtifactResolver.java:547)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:483)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:401)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:229)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:207)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifact(DefaultRepositorySystem.java:262)
	at org.jvnet.hudson.plugins.repositoryconnector.aether.Aether.resolve(Aether.java:101)`

Comment (jgangemi) [reply pom.xml:191]: interesting, that was removed b/c of it conflicted w/ some other version in the build and i'm using the latest version of the plugin.

can you let me know what your workflow looks like (are you using build params, just a plain artifact download)? do you have all there required plugin dependencies installed?

Comment (stifm) [reply pom.xml:191]: We currently have 102 jobs that use the plugin, with different combinations of Nexus build params and/or artifact downloads.
The error occured in two types of nightly builds before I reverted the plugin version:
1. artifact download that gets it's version parameter from a properties file on the file system, no build param involved
2. the same, but a build param with default value is used that is not relevant for artifact resolution

The dependant plugins are installed in version:
- Credentials Plugin 2.3.14
- Token Macro Plugin 2.14
The optional Struts Plugin is not installed.

Comment (stifm) [reply pom.xml:191]: The aforementioned jobs work with the new 2.0.4 release, so thanks for the update ;-)

**Pull request merged**